protocol/scripts/deploys/mainnet/v2/ramos/99-ramos-post-deploy.ts

```diff
@@ -0,0 +1,162 @@
+        RAMOS
+    ));
+
+    // Circuit breaker for Temple
+    await mine(TEMPLE_V2_INSTANCES.CORE.CIRCUIT_BREAKER_PROXY.setCircuitBreaker(
+        RAMOS_HASH,
+        TEMPLE_V2_ADDRESSES.CORE.TEMPLE_TOKEN,
+        TEMPLE_V2_ADDRESSES.STRATEGIES.RAMOS_STRATEGY.CIRCUIT_BREAKERS.TEMPLE
+    ));
+
+    // Circuit breaker for Dai
+    await mine(TEMPLE_V2_INSTANCES.CORE.CIRCUIT_BREAKER_PROXY.setCircuitBreaker(
+        RAMOS_HASH,
+        TEMPLE_V2_ADDRESSES.EXTERNAL.MAKER_DAO.DAI_TOKEN,
+        TEMPLE_V2_ADDRESSES.STRATEGIES.RAMOS_STRATEGY.CIRCUIT_BREAKERS.DAI
+    ));
+
+    // Grant permission for the circuit breaker proxy to call the underlying circuit breakers
+    await setExplicitAccess(
+        TEMPLE_V2_INSTANCES.STRATEGIES.RAMOS_STRATEGY.CIRCUIT_BREAKERS.DAI,
+        TEMPLE_V2_ADDRESSES.CORE.CIRCUIT_BREAKER_PROXY,
+        ["preCheck"],
+        true
+    );
+    await setExplicitAccess(
+        TEMPLE_V2_INSTANCES.STRATEGIES.RAMOS_STRATEGY.CIRCUIT_BREAKERS.TEMPLE,
+        TEMPLE_V2_ADDRESSES.CORE.CIRCUIT_BREAKER_PROXY,
+        ["preCheck"],
+        true
+    );
+}
+
+async function ramosPostDeploy(ramos: Ramos, TEMPLE_V2_ADDRESSES: ContractAddresses) {
+    await mine(ramos.setTokenVault(TEMPLE_V2_ADDRESSES.STRATEGIES.RAMOS_STRATEGY.ADDRESS));
+    await mine(ramos.setTpiOracle(TEMPLE_V2_ADDRESSES.TREASURY_RESERVES_VAULT.TPI_ORACLE));
+    await mine(ramos.setCoolDown(1800));
+    await mine(ramos.setRebalancePercentageBounds(100, 1_000));  // rebalance_up if 1% below, rebalance_down if 10% above
+    await mine(ramos.setMaxRebalanceAmounts(toAtto(1_000_000), toAtto(1_000_000), toAtto(1_000_000))); // 1Mill max on each
+    await mine(ramos.setPostRebalanceSlippage(5_000)); // 50%
+    await mine(ramos.setPoolHelper(TEMPLE_V2_ADDRESSES.RAMOS.TEMPLE_DAI.POOL_HELPER));
+
+    // The Automation EOA is allowed to call ramos.
+    await setExplicitAccess(
+        ramos,
+        TEMPLE_V2_ADDRESSES.RAMOS.AUTOMATION_EOA,
+        [
+            "rebalanceUpExit", // Unstake & single side withdraw TEMPLE, burn TEMPLE
+            "rebalanceDownJoin", // Mint TEMPLE, single side deposit, stake bpt
+        ],
+        true
+    );
+}
+
+async function stakingPostDeploy(auraStaking: AuraStaking, TEMPLE_V2_ADDRESSES: ContractAddresses) {
+    // RAMOS is allowed to call auraStaking.withdrawAndUnwrap() and auraStaking.depositAndStake()
+    await setExplicitAccess(
+        auraStaking,
+        TEMPLE_V2_ADDRESSES.RAMOS.TEMPLE_DAI.ADDRESS,
+        ["withdrawAndUnwrap", "depositAndStake"],
+        true
+    );
+
+    {
+        await auraStaking.setRewardsRecipient(
+            TEMPLE_V2_ADDRESSES.RAMOS.TEMPLE_DAI.FEE_COLLECTOR
+        );
+
+        await auraStaking.setAuraPoolInfo(
+            parseInt(TEMPLE_V2_ADDRESSES.RAMOS.TEMPLE_DAI.POOL_ID), 
+            TEMPLE_V2_ADDRESSES.RAMOS.TEMPLE_DAI.BPT_TOKEN, 
+            TEMPLE_V2_ADDRESSES.RAMOS.TEMPLE_DAI.REWARDS,
+        );
+    }
```

Comment:
Saw this was commented in the sepolia one, I had to add few extra contracts in `contract-addresses.ts` for this to work

Reply:
Yep correct - not required in testnet because we didn't have Aura set up, but it is required here